### ISSUE TITLE
Contrast errors on breadcrumbs

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#381]
To meet WCAG 2.0 AA standard contrast on images needs to be minimum 4.5:1.
Currently there are 4 errors

Steps to reproduce:
1. Download Wave extension for Chrome
2. Launch Wave tool when on document list page

![screen shot 2016-07-07 at 07 31 01](https://cloud.githubusercontent.com/assets/13200338/16644429/d3dc7a94-4414-11e6-830a-b134dec2f67d.png)

More information can be found at:
https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=143#visual-audio-contrast-contrast
http://webaim.org/standards/wcag/checklist#sc1.4.3

**Comment by VitoAlbano** [comment]
@hashraf88 Is this still on?
I've retested locally on development but it seems to be fine now , even the colors are changed

<img width="1142" alt="screen shot 2016-09-05 at 17 47 16" src="https://cloud.githubusercontent.com/assets/7974125/18253987/25d811a4-7391-11e6-8b59-fd11c9ca3645.png">

**Comment by hashraf88** [comment]
Yep this is fine now

**Comment by DenisVuyka** [comment]
Sorry for late update, the breadcrumb was reworked some time ago to take into account Material styles rather than former Bootstrap ones. Should be working fine now.

**Comment by VitoAlbano** [comment]
Ok i'll close the issue .  Thanks.
